test/build_payment.js

```diff
@@ -1,59 +1,50 @@
-const Client = require('../');
-const assert = require('assert');
-const fixtures = require('./fixtures');
+'use strict';
 
-describe('Ripple REST Client buildPayment', function() {
-  var client;
-
-  before(function () {
-    client = new Client({
-      account: fixtures.ripple_address.source_account
-    });
-  });
-
-  it('should successfully build payment', function(done){
-
-    var newPayment = {
-      currency: 'GWD',
-      amount: 10,
-      recipient: fixtures.ripple_address.destination_account,
-      source_currencies: ['GWD'],
-      from_issuer: fixtures.ripple_address.destination_account,
-      to_issuer: fixtures.ripple_address.destination_account
-    };
+var Client = require('../');
+var assert = require('assert');
+var account_info = require('./fixtures/account_info')();
+var common = require('./common/common.js');
+var fixture = require('./fixtures/build_payment');
 
-    client.buildPayment(newPayment, function(error, response){
-      assert(!error);
-      assert.deepEqual(fixtures.responses['1_4_0'].success.buildPayment, response);
-      done();
+describe('Ripple REST Client buildPayment', function() {
+  var client = new Client({
```

Comment:
Instead of creating this client before the tests and running the tests against it. Create the `Client` in a `beforeEach` and clean it up in a `afterEach`. That way any sideeffects that affect the `Client` won't bleed through in other tests.